lib/dom.js

```diff
@@ -1,3 +1,5 @@
+"use strict";
```

Comment:
I don't have a clear understanding of the impact, might this be a breaking change?
According to MDN "[Strict mode changes both syntax and runtime behavior.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Changes_in_strict_mode)".
So I would rather wait for more tests to be available before making such a change.

Reply:
It's possible to statically determine all of the new errors strict mode might throw. Iirc, eslint raises errors for all of these cases by default.